packages/dynamic-import-vars/src/index.js

```diff
@@ -55,6 +55,12 @@ function dynamicImportVariables({ include, exclude, warnOnError } = {}) {
               r.startsWith('./') || r.startsWith('../') ? r : `./${r}`
             );
 
+            if (errorWhenNoFilesFound && paths.length === 0) {
+              this.error(
+                new Error('No files found when trying to dynamically load concatted string')
```

Comment:
This error could do with a bit more context. What directory we are expecting there to be files for example.